### PR TITLE
HoF & Update

### PR DESCRIPTION
**Pull request by 2009Legacy**
This is just a quick addition/cleanup to the new's post. I have added a x10 PDIM HoF & Updated the x5 HoF.

### CODE REVIEW

File: site/community/hall_of_fame_x10.html [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+layout: guide
+---
+<div id="content">
+<div id="article">
+<div class="sectionHeader">
+<div class="left">
+<div class="right">
+<h1 class="plaque">
+
+                                    Hall of Fame
```
Comment (ryannathans) [inline site/community/hall_of_fame_x10.html:13]: funny spacing

File: site/community/hall_of_fame.html [hunk 2]
```diff
@@ -48,12 +61,18 @@ <h1 class="plaque">
...
-

+<div class="helptitle"><b>Important Information</b></div>
+
+This entire website is getting a massive overhaul and this page will change. If you need any help at all ask a staff member either on our 
+
+<a href="https://discord.gg/43YPGND">Discord</a> or if you do not prefer discord check out our <a href="https://matrix.to/#/#09scape-general:matrix.org">Matrix</a>!
+
+
+
```
Comment (ryannathans) [inline site/community/hall_of_fame.html]: white space be thicc

File: site/community/hall_of_fame_x10.html [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+<br/>
+<br/>
+<br/>
+</div>
+<br/>
+<br/>
+<section style="margin-left: 5%;">
+<div class="bold">Further Help</div>
+
+                                                                        Need more help? Come chat with us!
```
Comment (ryannathans) [inline site/community/hall_of_fame_x10.html]: why it do be like this?

Comment (2009Legacy) [reply site/community/hall_of_fame_x10.html]: the editor

**Pull request merged**